utils/resultstable.go

```diff
@@ -1057,6 +1060,9 @@ func getFinalApplicabilityStatus(applicabilityStatuses []jasutils.ApplicabilityS
 	foundUndetermined := false
 	foundNotCovered := false
 	for _, status := range applicabilityStatuses {
+		if status == jasutils.MissingContext {
+			return jasutils.MissingContext
+		}
```

Comment:
Missing Context does not have the first priority.
If we have a case of cve with MissingContext and another Cve with Applicable. we want to return the Applicable. only if not Applicable you can return the `MissingContext`, make it similar logic to `ApplicabilityUndetermined`